CHANGE_LOG

```diff
@@ -1,3 +1,54 @@
+v0.44.0rc1 (Nov 8, 2024)
+------------------------
```

Comment:
```suggestion
v0.44.0rc1 (November 8, 2024)
-----------------------------
```
sorry, one last minor thing for consistency in the change log, month names are not abbreviated.